alma/lib/Utils/CustomerHelper.php

```diff
@@ -0,0 +1,199 @@
+                'payment_method' => 'alma',
+                'shipping_method' => $carrierData->getNameById($this->cart->id_carrier),
+                'items' => CartData::cartItems($this->cart),
+            ],
+            'previous_orders' => [
+                $customerOrderHelper->previous($this->customer->id),
+            ],
+        ];
+    }
+
+    /**
+     * Get birthday date
+     *
+     * @return string|null
+     */
+    private function getBirthday()
+    {
+        $birthday = $this->customer->birthday;
+        if ($birthday == '0000-00-00') {
+            return null;
+        }
+
+        return $birthday;
+    }
+
+    /**
+     * Get addresses of customer
+     *
+     * @return array getAddressesData
+     */
+    private function getAddressesData()
+    {
+        $addresses = [];
+
+        foreach ($this->getCustomerAddresses() as $address) {
+            array_push($addresses, [
+                'line1' => $address['address1'],
+                'postal_code' => $address['postcode'],
+                'city' => $address['city'],
+                'country' => Country::getIsoById((int) $address['id_country']),
+                'county_sublocality' => null,
+                'state_province' => $address['state'],
+            ]);
+        }
+
+        return $addresses;
+    }
+
+    /**
+     * Get addresses of customer Prestashop
+     *
+     * @return array getCustomerAddresses
+     */
+    private function getCustomerAddresses()
+    {
+        if (version_compare(_PS_VERSION_, '1.5.4.0', '<')) {
+            $idLang = $this->context->language->id;
+        } else {
+            $idLang = $this->customer->id_lang;
+        }
+
+        return $this->customer->getAddresses($idLang);
+    }
+
+    /**
+     * get phone between info customer and addresses customer
+     *
+     * @return string
+     */
+    private function getPhone()
+    {
+        $phone = null;
+        $addressData = new AddressData($this->cart);
+        $shippingAddress = $addressData->getShippingAddress();
+
+        if ($shippingAddress->phone) {
+            $phone = $shippingAddress->phone;
+        } elseif ($shippingAddress->phone_mobile) {
+            $phone = $shippingAddress->phone_mobile;
+        }
+
+        if (is_null($phone)) {
+            foreach ($this->getCustomerAddresses() as $address) {
+                if ($address['phone']) {
+                    $phone = $address['phone'];
+                } elseif ($address['phone_mobile']) {
+                    $phone = $address['phone_mobile'];
+                }
+            }
+        }
+
+        return $phone;
+    }
+
+    /**
+     * If is new customer or not
+     *
+     * @param int $idCustomer
+     *
+     * @return bool
+     */
+    private function isNew($idCustomer)
+    {
+        if (Order::getCustomerNbOrders($idCustomer) > 0) {
+            return false;
+        }
+
+        return true;
```

Comment:
Try to simplify your code by avoiding `if` statement when your method should return a `bool` (I am not sure here about the `getCustomerNbOrders` return type - `string`, `int`, `mixed`, `null` ? - and the usage of `===`)

```suggestion
        return Order::getCustomerNbOrders($idCustomer) === 0;
```